src/test/munit/all-validation-test-suite.xml

```diff
@@ -41,4 +40,28 @@
         </munit:validation>
     </munit:test>
 
+    <munit:test name="validationAllWithErrorMappingResolvingToMuleUnknown"
+                description=""
+                tags="MULE-19139"
+                ignore="#[Munit::muleVersionPriorTo('4.2.2') or Munit::muleVersionEqualTo('4.2.2')]">
```

Comment:
why not just `#[Munit::muleVersionPriorTo('4.2.3')]`

Reply:
4.2.3 would leave out 4.2.3-SNAPSHOT, so I changed it to 4.2.3-SNAPSHOT